### ISSUE TITLE
Repo Organization

### ISSUE THREAD

**Issue by a-vanderheiden** [adv-py-geos/Easterwood_rain#21]
I was thinking that in general, we can file all of our .py files away into subdirectories, and have a couple example notebooks in the root directory to use for demonstration of the package. 

I'll get working on consolidating and organizing the .py files and making an ```__init__.py``` file. 

So far, the only function I see that gets called from our package is `create_df()` from `readdata.py`. Are there any others that we currently use? If there are others, please leave a comment so I can include it in the ```__init__.py``` and not screw up the references to our functions.

**Comment by samanthalongridge** [comment]
I like the idea of putting the .py files in subdirectories and having example notebooks in the root directory, do you have any suggestions for different subdirectories we should have?

**Comment by a-vanderheiden** [comment]
AH, so sorry it took me so long to get back. I'm thinking we can just chuck all the .py files into one folder from now because we don't have too much going on. Im working on it currently and so far I just have the folder saved as Easterwood_rain again following the xroms package as an example. 

I'm also condensing the number of .py files. For example, there are several scripts with a single function that makes a plot from the data. So, I am going to put all the individual functions into the single plot_util.py file

**Comment by a-vanderheiden** [comment]
I just made a pull request with some of these above changes if you would like to check it out! I haven't consolidated the various plotting .py files yet; that will come later this weekend.

**Comment by samanthalongridge** [comment]
Okay, I think I'll try to put some of jupyter notebook files all in one place and leave the machine learning notebooks in the root directory.

**Comment by a-vanderheiden** [comment]
Yeah I think that would help clean up the repo! How were you thinking of doing it? 

I was playing around with the idea of having a folder where everyone can workshop ideas in however many notebooks they want, but then have a "show pony" notebook in the root directory where we have a prediction example for each of the methods everyone has made so far.

**Comment by samanthalongridge** [comment]
I was just going to put the stats and preliminary plots that are .ipynb files together (the first steps that we all did before we made them .py files) into a folder.

**Comment by dylanschlichting** [comment]
This all sounds good. One suggestion I have once we've finished the organization is to update the readme file so that it nicely summarizes the repo and acts like a sort of 'abstract'. For example, something less sophisticated than xroms: https://github.com/kthyng/xroms.
Since y'all have taken the initiative on the organization - I could throw together the readme.

**Comment by a-vanderheiden** [comment]
That would be awesome if you could throw that together!
